components/feature/addons/src/main/res/layout/mozac_feature_addons_unsupported_item.xml

```diff
@@ -5,40 +5,46 @@
 
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="horizontal"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:gravity="center_vertical"
+    android:orientation="horizontal"
     android:padding="16dp">
 
     <ImageView
         android:id="@+id/add_on_icon"
         android:layout_width="48dp"
         android:layout_height="48dp"
-        android:src="@drawable/mozac_ic_extensions"
         android:importantForAccessibility="no"
-        android:scaleType="fitCenter" />
+        android:scaleType="fitCenter"
+        android:src="@drawable/mozac_ic_extensions"
+        android:tint="?android:attr/textColorPrimary" />
 
-    <TextView
-        android:id="@+id/add_on_name"
+    <LinearLayout
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textStyle="bold"
-        android:textSize="16sp"
-        tools:text="uBlock Origin" />
+        android:layout_marginStart="16dp"
+        android:orientation="horizontal">
 
-    <View
-        android:layout_width="0dp"
-        android:layout_height="1dp"
-        android:layout_weight="1" />
+        <TextView
+            android:id="@+id/add_on_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="18sp"
+            tools:text="uBlock Origin" />
 
-    <ImageButton
-        android:id="@+id/add_on_remove_button"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:background="@null"
-        android:tint="@android:color/darker_gray"
-        android:contentDescription="@string/mozac_feature_addons_remove"
-        android:src="@drawable/mozac_ic_delete" />
+        <View
+            android:layout_width="0dp"
+            android:layout_height="1dp"
+            android:layout_weight="1" />
 
+        <ImageButton
+            android:id="@+id/add_on_remove_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@null"
+            android:contentDescription="@string/mozac_feature_addons_remove"
+            android:src="@drawable/mozac_ic_delete"
+            android:tint="@android:color/darker_gray" />
```

Comment:
Not sure if this color is going to adapt to the Fenix night and light theme, would you mind try it?